site/community/release_guide.md

```diff
@@ -234,21 +244,14 @@ Use Maven release plugin to create the release branch and update the current bra
 > $ git reset --hard apache/<master branch OR release tag>
 > $ git branch -D ${BRANCH_NAME}
 
-However, if you are doing an incremental/hotfix release, please run the following command after checking out the release tag of the release being patched.
```

Comment:
what to do for minor release?

Reply:
@sijie I thought that "incremental/hotfix" is a synonym for "minor"

Reply:
yes. but my question is why do you removing the command below. They should be different commands, no?

Reply:
for a minor (4.5.2) release you do not have to use that command, it will create a new branch, but it this case the branch is already here (branch-4.5 in my case)

Reply:
okay make sense